dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -439,41 +495,37 @@ def test_histogram_with_varying_number_of_bin(self):
         profiler2.max_histogram_bin = 10000
         profiler2.update(df2)
         num_bins = len(profiler2.profile['histogram']['bin_counts'])
-        self.assertEqual(num_bins, 10000)
+        self.assertEqual(num_bins, 6)
```

Comment:
it's supposed to be a very large number of bins in this case, right?

Reply:
fixed